### ISSUE TITLE
Avoid localhost clashes

### ISSUE THREAD

**Issue by dushoff** [ICI3D/ICI3D.github.io#36]
I suggest that the different repos and branches use different ports, not starting right at 4000, to make it easier to run multiple jekyll instances

**Comment by pearsonca** [comment]
ici3d root will be [2049](https://www.imdb.com/title/tt1856101/)

**Comment by jrcpulliam** [comment]
Seems like DAIDD is 4167 and MMED is 4000; so is this not taken care of? If so, please close

**Comment by dushoff** [comment]
No, it seems OK for them to all be different. I try to record the standard in the Makefile, but maybe the README should say it (or point to the file where it's really recorded). 4000 is not ideal (since it's the default) -- the idea is that you can just run jekyll without worrying about whether it will conflict with another jekylly on your machine.
